helm/dashboards/charts/public_dashboards/dashboards/shared/public/servicemonitors-details.json

```diff
@@ -0,0 +1,565 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "ServiceMonitors details: servicemonitors drilldown",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 1,
+  "id": 135,
+  "links": [],
+  "panels": [
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 5,
+      "panels": [],
+      "title": "Help",
+      "type": "row"
+    },
+    {
+      "datasource": {
```

Comment:
Can we talk about the datasource again? :p

Reply:
This is the text panel, I don't understand why it has a datasource :shrug: 
Anyway, the panels that actually do queries should all have the variabel datasource :grin:

Reply:
I'm fine with it, but dashboard linter will not :D